Library/Formula/fontconfig.rb

```diff
@@ -8,7 +8,7 @@ class Fontconfig < Formula
   # is relocatable, but it has hardcoded paths in the executables.
   bottle do
     cellar :any
-    revision 3
+    revision 4
```

Comment:
Ahem no sorry don’t bump anything the bot will bump the bottles when we’ll merge the PR.